cohort-engine-api-web/src/main/resources/logback.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--  ************************************** -->
+<!--  (C) Copyright IBM Corp. 2001, 2020     -->
+<!--  SPDX-License-Identifier: Apache-2.0    -->
+<!--  ************************************** -->
+<configuration>
+	<appender name="CUSTOM"
+		class="com.ibm.watson.common.service.base.CustomAppender">
+		<encoder>
+			<pattern>%m%n</pattern>
+		</encoder>
+	</appender>
+
+	<root level="info">
+		<appender-ref ref="CUSTOM" />
+	</root>
+</configuration>
```

Comment:
This is mostly just a comment. I jump through hoops in cohort-parent/pom.xml to remove transitive dependencies on logback from upstream OSS components because I don't believe that consumers of our library should be bound to a particular logging implementation and SLF4J likes to complain when you have more than one binding in your classpath. 

That said, I have no problem with the REST API layer being dependent on logback since it is functionally an app and nothing will link to it. I just want to make sure the logback dependency doesn't leak back into the library level code. I don't see that happening. Logback is never referenced in our pom's, so we must just be getting it transitively from the framework.

Reply:
The logback dependency is something provided by the rest service framework and I copied it in based on how the example application used it. It is new since the last time I used the framework and I will not claim to be an expert in it. So your comments are welcome, we should keep an eye out to see if it causes any problems once we start calling the cohort engine jar from the rest api, as I am currently packaging the cohort jar in the .war, but not yet calling it.